osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -665,4 +492,24 @@ public enum ScoringMode
         [LocalisableDescription(typeof(GameplaySettingsStrings), nameof(GameplaySettingsStrings.ClassicScoreDisplay))]
         Classic
     }
+
+    [Serializable]
+    [MessagePackObject]
+    public class ScoreProcessorStatistics
```

Comment:
This is quite a bit of extra data to be sending in every score frame. I think it's going to probably double the bandwidth requirement of spectator (it was already pretty high so maybe a concern?)

Reply:
It's every 100 frames/when they get purged. Do you have a solution in mind?

Reply:
I think I got confused by the class name `FrameHeader` and https://github.com/ppy/osu/blob/f8830c6850128456266c82de83273204f8b74ac0/osu.Game/Rulesets/Replays/ReplayFrame.cs#L18-L23

Maybe it should be renamed to `FrameBundleHeader` or something...